security/rbac/README.md

```diff
@@ -133,6 +133,30 @@ confluent iam rolebinding create --principal User:$CLIENT --role ResourceOwner -
 # REST Proxy Admin: no additional administrative rolebindings required because REST Proxy just does impersonation
 
 # Producer/Consumer
- confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Topic $TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID
- confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Group $CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID
+confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Topic $TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID
+confluent iam rolebinding create --principal User $CLIENTB --role ResourceOwner --resource Group $CONSUMER_GROUP --kafka-cluster-id $KAFKA_CLUSTER_ID
+```
+
+
+## KSQL
+
+* [delta_configs/ksql-server.properties.delta](delta_configs/ksql-server.properties.delta)
+* Role bindings:
+
+```bash
+# KSQL Server Admin
+confluent iam rolebinding create --principal User:$ADMIN_KSQL --role ResourceOwner --resource Topic:_confluent-ksql-${KSQL_SERVICE_ID}_command_topic --kafka-cluster-id $KAFKA_CLUSTER_ID
+confluent iam rolebinding create --principal User:$ADMIN_KSQL --role ResourceOwner --resource Topic:${KSQL_SERVICE_ID}ksql_processing_log --kafka-cluster-id $KAFKA_CLUSTER_ID
+confluent iam rolebinding create --principal User:$ADMIN_KSQL --role SecurityAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID --ksql-cluster-id $KSQL_SERVICE_ID
+confluent iam rolebinding create --principal User:$ADMIN_KSQL --role ResourceOwner --resource KsqlCluster:ksql-cluster --kafka-cluster-id $KAFKA_CLUSTER_ID --ksql-cluster-id $KSQL_SERVICE_ID
+
+# KSQL CLI queries
+confluent iam rolebinding create --principal User:${KSQL_USER} --role ResourceOwner --resource KsqlCluster:ksql-cluster --kafka-cluster-id $KAFKA_CLUSTER_ID --ksql-cluster-id $KSQL_SERVICE_ID
```

Comment:
@spena -- goal is to be as restrictive as possible, i.e., to provide the minimum access possible.  If `ResourceOwner` is too permissive, it is not because there is a use case, it is because it's a mistake.  Which specific commands should be DeveloperWrite (versus ResourceOwner), or is it all of them for `KSQL_USER`?

Reply:
I have updated this to `DeveloperWrite` in https://github.com/confluentinc/examples/pull/204/commits/bd31933c635e0d4fc6817a6d63d33a2a319a744d

Thanks!